pkg/controller/support/credentials/cloudauth/ecr/ecr_test.go

```diff
@@ -1,133 +1,198 @@
-//go:build integration
-// +build integration
-
 package ecr
 
 import (
 	"context"
 	"errors"
+	"fmt"
 	"reflect"
-	"strings"
 	"testing"
 
-	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/service/ecr"
-	"github.com/aws/aws-sdk-go-v2/service/ecr/types"
-	dockertypes "github.com/docker/docker/api/types"
-	"github.com/go-logr/logr"
-
-	"github.com/dominodatalab/hephaestus/pkg/controller/support/credentials/cloudauth"
+	ecrTypes "github.com/aws/aws-sdk-go-v2/service/ecr/types"
+	dockerTypes "github.com/docker/docker/api/types"
+	"github.com/go-logr/zapr"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zaptest/observer"
+	"gotest.tools/v3/assert"
```

Comment:
This is the wrong `assert` import. You'll notice that this added the line https://github.com/dominodatalab/hephaestus/pull/67/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R55 to your `go.mod`.

You want `"github.com/stretchr/testify/assert"`